internal/proxy/usecases/proxy_usecase.go

```diff
@@ -63,24 +65,29 @@ func NewProxyUsecase(
 		rollingPatterns:  setupRegexpFor(config.ProxyConfig.Proxy.WhitelistedRolling),
 		baseArchiveURL:   baseArchiveURL,
 		baseRollingURL:   baseRollingURL,
+		network:          config.ProxyConfig.Tezos.Network,
 	}
 }
 
-func (p *ProxyUsecase) findInDatabaseIfNotFoundInCache(UUID string) error {
-	_, err := p.projectCacheRepo.FindByUUID(UUID)
+func (p *ProxyUsecase) findInDatabaseIfNotFoundInCache(UUID string) (*pkgmodel.Project, error) {
+	cachePrj, errCache := p.projectCacheRepo.FindByUUID(UUID)
```

Comment:
I dont understand why are you doing a difference between errCache and err.
If I remember well my golang, err is a naming convention in golang (not trivial to understand I agree - like a lot of things in golang  - but if I remember well this is the case)

Reply:
I prefer to keep it for code readability (when debbugging unit test)